src/components/SearchForm.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="wrapper">
+    <p>
+      <span class="text_netflix">netflix</span>
+      <span class="text_routlette">roulette</span>
+    </p>
+    <h2>{{ headerText }}</h2>
+    <form class="search-form" @submit="handleSubmit" :style="style">
+      <div class="search-wrapper">
+        <input
+          class="search-form__input"
+          type="text"
+          name="searchValue"
+          placeholder="Search..."
+          v-model="searchValue"
+        />
+        <input type="submit" value="Search" class="search-form__submit" />
+      </div>
+      <SwitcherComponent
+        v-on:switcher-data="handleSwitcher"
```

Comment:
@switcherData is the same, let's use @ instead of v-on or vice versa, but only one of them